apisix/discovery/consul/init.lua

```diff
@@ -225,8 +225,8 @@ local function watch_catalog(consul_server)
     ::RETRY::
     local watch_result, watch_err = client:get(consul_server.consul_watch_catalog_url)
     local watch_error_info = (watch_err ~= nil and watch_err)
-                             or ((watch_result ~= nil and watch_result.status ~= 200)
-                             and watch_result.status)
+                            or ((watch_result ~= nil and watch_result.status ~= 200)
+                            and watch_result.status)
```

Comment:
This appears to be a code style modification only. If it does not contain substantive changes, please leave it as is.